src/components/Event/EventItemView/EventItemView.jsx

```diff
@@ -150,205 +150,201 @@ export default class EventItemView extends Component {
     const canDeleted = isMyEvent && eventStatus === EVENT_STATUS_ENUM.CANCELED;
 
     return (
-      <>
-        <div className="container-fluid mt-1">
-          <div className="row">
-            <div className="col-9">
-              <div className="col-12">
-                <img
-                  src={this.state.eventImage}
-                  id={`eventFullPhotoImg${id}`}
-                  alt="Event"
-                  className="w-100"
-                />
-                <div className="text-block">
-                  <span className="title">{title}</span>
-                  <br />
-                  {isPublic ? (
-                    <span>{EVENT_ITEM_VIEW_CONSTS.PUBLIC_EVENT}</span>
-                  ) : (
-                    <span>{EVENT_ITEM_VIEW_CONSTS.PRIVATE_EVENT}</span>
-                  )}
-                  <br />
-                  {maxParticipants < INT32_MAX_VALUE ? (
-                    <span className="maxParticipants">
-                      {visitorsEnum.approvedUsers.length}
-                      {"/"}
-                      {maxParticipants}
-                      <span className="pl-2">
-                        {EVENT_ITEM_VIEW_CONSTS.PARTICIPANTS}
-                      </span>
+      <div className="container-fluid mt-1">
+        <div className="row">
+          <div className="col-9">
+            <div className="col-12">
+              <img
+                src={this.state.eventImage}
+                id={`eventFullPhotoImg${id}`}
+                alt="Event"
+                className="w-100"
+              />
+              <div className="text-block">
+                <span className="title">{title}</span>
+                <br />
```

Comment:
maybe <br /> tag using in tags as line break element>?

Reply:
inline span